src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    protected static final String[] colors = new String[]{"blue", "green", "red", "white"};
```

Comment:
если ты нигде больше не используешь константу COLORS, то модификатор доступа лучше делать private
```suggestion
    protected static final String[] COLORS = new String[]{"blue", "green", "red", "white"};
```